pages/user.js

```diff
@@ -1,159 +1,174 @@
+              </ul>
+            </div>
+        )))}
+      </div>
+      <SimpleForm
+        name="signClaim"
+        handleSubmit={handleSubmit}
+        meta={{
+          inputs: [{
+            name: 'msg',
+            label: 'message',
+          }],
+          submit: {
+            text: 'Sign & Attach',
+          },
+        }}
+      />
+      <SimpleForm
+        name="sendMessage"
+        handleSubmit={handleMamSubmit}
+        meta={{
+          inputs: [{
+            name: 'msg',
+            label: 'sendMamMessage',
+          }],
+          submit: {
+            text: 'Send',
+          },
+        }}
+      />
+
+      <div style={{ display: 'flex', justifyContent: 'center' }}>
+        <Button onClick={handleSetUser} type="submit" variant="raised" color="primary" size="small">
+          SetUser
+        </Button>
+      </div>
+    </Layout>
+  );
+};
 
-			<div style={{display: 'flex', justifyContent: 'center'}}>
-				<Button onClick={handleSetUser} type="submit" variant="raised" color="primary" size="small">
-                   SetUser
-                </Button>
-			</div>
-		</Layout>
-	)
-}
+UserPage.propTypes = {
+  claims: PropTypes.array,
+  messages: PropTypes.object,
+  user: PropTypes.object,
+  createClaim: PropTypes.func,
+  createMamMessage: PropTypes.func,
+};
 
 UserPage.getInitialProps = async (context) => {
-	const { store } = context
-	const { id } = context.query
-//	await store.dispatch(fetchOffTangleUserList())
-        await store.dispatch(fetchUserList())
-        /* what is this used for ? */
-	await store.dispatch(checkTangleUsers([{ id }]))
-	await store.dispatch(fetchClaims(id))
-	await store.dispatch(fetchMamMessages(id))
-	const { users } = store.getState()
-	const { localList, validData, claims, messages } = users
-	/* not working at all */
-	const validIds = validData.map(v => v.id)
-	// TODO: Use selector to get better performance
-//	const userList = offTangleData.filter(d => validIds.indexOf(d.id) !== -1)
-	const userList = localList//.filter(d => validIds.indexOf(d.id) !== -1)
-	const user = userList.find(u => u.id === id)
-	const otherUser = userList.map(u => {return u.id}).filter(i => i !== id)
-	const { pk } = user
-	user.qrcode = await transformToQRCode(JSON.stringify({pk, id}))
-	return {
-		id,
-		user,
-		claims,
-		messages,
-		otherUser,
-	}
-}
+  const { store } = context;
+  const { id } = context.query;
+  //  await store.dispatch(fetchOffTangleUserList())
+  await store.dispatch(fetchUserList());
+  /* what is this used for ? */
+  await store.dispatch(checkTangleUsers([{ id }]));
+  await store.dispatch(fetchClaims(id));
+  await store.dispatch(fetchMamMessages(id));
+  const { users } = store.getState();
+  const {
+    localList, claims, messages,
+  } = users;
+  /* not working at all */
+  // const validIds = validData.map(v => v.id);
+  // TODO: Use selector to get better performance
+  //  const userList = offTangleData.filter(d => validIds.indexOf(d.id) !== -1)
```

Comment:
@HexRabbit, Are these comments  necessary? 🤔

Reply:
No, they're unnecessary. It's the code I've deleted from the first version.

Reply:
Good. Let's get rid of such kludge.